src/utils/math/transformNumeric.spec.ts

```diff
@@ -4,20 +4,15 @@ test('Suffixes numeric values with "px" string', () => {
   expect(transformNumeric(5)).toEqual('5px')
 })
 
-test('Returns string values as is', () => {
+test('Returns string values as it is', () => {
```

Comment:
There is nothing wrong with "as is" grammar construction ;)

Reply:
my bad

Reply:
reverted